### PR TITLE
Docs: Migration guide - schema configuration

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

Add additional helpful information to the migration guide regarding the schema configuration.
Creds to @jharris4!

### CODE REVIEW

**Comment by jharris4** [comment]
🚀

File: docs/modules/ROOT/pages/guides/migration-guide/server.adoc [hunk 1]
```diff
@@ -30,6 +30,55 @@ const schema = await neo4jGraphQL.getSchema();
...
+To achieve the same behaviour with the new `@neo4j/graphql` library, the GraphQL schema has to be extended instead. Note, no additional configuration or parameters need to be passed to the `getSchema()` function.
+To follow the example above, in the  `@neo4j/graphql` library this can be achieved the following way for the first case:
+[source, graphql, indent=0]
+----
+type NameOfType @exclude(operations: [CREATE, UPDATE, DELETE]) {
+  name: String
+}
+----
+
+Or like this for the second case:
```
Comment (mjfwebb) [inline docs/modules/ROOT/pages/guides/migration-guide/server.adoc:71]:
```suggestion
Or to exclude queries for a specific type like so:
```

**Review by mjfwebb — CHANGES_REQUESTED** [review]
Nice! Small suggestions to make it clearer which examples are related (and for search-ability).

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-1507.surge.sh

**Review by mjfwebb — DISMISSED** [review]

**Pull request merged**